<?xml version="1.0"?>
<robot name="sp1_robot">
  <link name="base_link">
    <inertial>
      <origin xyz="0 0 0" rpy="0 0 0"/>
      <mass value="353.00"/> <!-- Mass in kilograms -->
      <inertia ixx="0.163" ixy="-0.171" ixz="-3.391E-07"
               iyy="0.178" iyz="-2.773E-07"
               izz="0.342"/> <!-- Moments of inertia in kg*m^2 -->
    </inertial>
    <visual>
      <origin xyz="0 0 0" rpy="0 0 0"/>
      <geometry>
        <mesh filename="file:///home/akshay/ros2_ws/virtual_shake_robot_pybullet/models/SP1_PBRmodel/SP1_az_1.obj" scale="1.0 1.0 1.0"/>
      </geometry>
      <material name="grey">
        <color rgba="0.5 0.5 0.5 1"/>
      </material>
    </visual>
    <collision>
      <origin xyz="0 0 0" rpy="0 0 0"/>
      <geometry>
        <mesh filename="file:///home/akshay/ros2_ws/virtual_shake_robot_pybullet/models/SP1_PBRmodel/SP1_az_1.obj" scale="1.0 1.0 1.0"/>
      </geometry>
    </collision>
  </link>
</robot>
